src/UI/IconState/index.tsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { IconProp } from '@fortawesome/fontawesome-svg-core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import classNames from 'classnames';
-import globalStyles from 'assets/sass/main.scss';
+
 import { WithClassnameType } from '../types';
 
-export interface IconStateProps extends WithClassnameType {
+import globalStyles from 'assets/sass/main.scss';
+
+export interface IconStatePropsType extends WithClassnameType {
```

Comment:
this is a braking change because IconStateProps was exported before. Let's make breaking changes with the next release

Reply:
Reverted and commented. 👍